tests/linux/test_backend_utils.cpp

```diff
@@ -67,8 +67,8 @@ void simulate_qemuimg_info(const mpt::MockProcess* process, const QString& expec
         ON_CALL(*process, read_all_standard_error).WillByDefault(Return(produce_output));
 }
 
-void simulate_qemuimg_resize(const mpt::MockProcess* process, const QString& expect_img,
-                             const mp::MemorySize& expect_size, const mp::ProcessState& produce_result)
+void simulate_qemuimg_resize(mpt::MockProcess* process, const QString& expect_img, const mp::MemorySize& expect_size,
```

Comment:
Intriguing that this worked before. I suppose actions go outside of the mock...

Reply:
Yup.